src/Integration.UnitTests/Telemetry/TelemetryManagerTests.cs

```diff
@@ -156,47 +157,15 @@ public void WhenFirstCallDelayerAndNewDay_ChangeLastUploadAndSaveAndSendPayload(
                 {
                     NumberOfIssuesInvestigatedRemotely = 777,
                     NumberOfIssuesInvestigatedLocally = 888
-                }
-            };
-            telemetryRepositoryMock.Setup(x => x.Data).Returns(telemetryData);
-
-            var manager = CreateManager();
-            var now = DateTimeOffset.Now;
-
-            // Act
-            telemetryTimerMock.Raise(x => x.Elapsed += null, new TelemetryTimerEventArgs(now));
-
-            // Assert
-            telemetryData.LastUploadDate.Should().Be(now);
-            telemetryData.Analyses.Count.Should().Be(0); // should have cleared the list of installed languages
-            telemetryData.ShowHotspot.NumberOfRequests.Should().Be(0); // should have cleared the counter
-            telemetryData.TaintVulnerabilities.NumberOfIssuesInvestigatedRemotely.Should().Be(0); // should have cleared the counter
-            telemetryData.TaintVulnerabilities.NumberOfIssuesInvestigatedLocally.Should().Be(0); // should have cleared the counter
-            telemetryRepositoryMock.Verify(x => x.Save(), Times.Once);
-            telemetryClientMock.Verify(x => x.SendPayloadAsync(It.IsAny<TelemetryPayload>()), Times.Once);
-        }
-
-        [TestMethod]
-        public void WhenTryUploadDataTimerElapsedAndNewDay_ChangeLastUploadAndSaveAndSendPayload()
```

Comment:
This test was exactly the same as `WhenFirstCallDelayerAndNewDay_ChangeLastUploadAndSaveAndSendPayload`